.github/CODEOWNERS

```diff
@@ -98,99 +97,36 @@
 # ServiceLabel: %AI Projects
 # ServiceOwners:                                                   @dargilco @jhakulin @nick863
 
-# ServiceLabel: %AKS
```

Comment:
Why are we doing anything beyond changing the headings?    Please explain why I'm seeing bit swaths of removes and adds.